.circleci/config.yml

```
@@ -97,10 +97,18 @@ jobs:
           command: make unit-tests
           environment:
             TEST_RESULTS_DIR: workspace/test-results
+      - install-coverage
+      - run:
+          name: Generate Coverage
+          command: |
+            coverage json --data-file="workspace/test-results/.coverage.unit" \
+            -o workspace/test-results/coverage_unit.json
       - store_test_results:
           path: workspace/test-results
       - store_artifacts:
-          path: workspace/test-results
+          path: workspace/test-results/coverage.json
+      - store_artifacts:
+          path: workspace/test-results/unit_results.xml
```

The file name should be coverage_unit.json?

```suggestion
          path: workspace/test-results/coverage.json
      - store_artifacts:
          path: workspace/test-results/unit_results.xml
```

_Optional_ Just a thought but if you changed the files to output in an `artifacts` directory, you could simplify this step I think.
```suggestion
          path: workspace/test-results/artifacts
```

L99
```
          environment:
            TEST_RESULTS_DIR: workspace/test-results/artifacts
```

L105
```
          command: |
            coverage json --data-file="workspace/test-results/.coverage.unit" \
            -o workspace/test-results/artifacts/coverage_unit.json
```

---

`path: workspace/test-results/artifacts` would include `.coverage` file as well

---

oh! i thought it would be different because we specify `--data-file="workspace/test-results/.coverage.unit` 🤷🏻‍♀️ I guess we shall have two steps then 😸